LFS171x/sawtooth-material/sawtooth-tuna/client/src/state.js

```diff
@@ -1,5 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
 
+/* Contributed by Zac Delventhal @delventhalz 
```

Comment:
This code was written by Zac Delventhal @delventhalz.

Reply:
rephrased